tools/kdc/setup_kdc.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Should we add a `set -e` here so that this fails fast?